### DISCUSSION TITLE
The property jedec-id is required by the jedec,spi-nor driver, but is not listed as such in jedec,spi-nor.yaml

### DISCUSSION THREAD

**Discussion by hieuv** [zephyrproject-rtos/zephyr#69299] [RFC]
The property `jedec-id` is required by the jedec,spi-nor driver in drivers/flash/spi_nor.c, but is not listed as such in jedec,spi-nor.yaml.

I am not an expert on the driver side, but this seems like a bug. 
I was opening a Bug Report, but met a note saying that if I am not sure, maybe start here.

Can someone more proficient with drivers confirm whether this is a bug?

**Comment by GeorgeCGV** [comment]
Looks like it is only required for `CONFIG_SPI_NOR_SFDP_RUNTIME=n` configuration.
